### DISCUSSION TITLE
peak areas and intensities are much lower than when looking at them in SCIEXOS

### DISCUSSION THREAD

**Discussion by Lipidbender** [systemsomicslab/MsdialWorkbench#323] [General]
When  processing WIFF2 files from Sciex Instruments in MSDIAL the peak areas and intensities are much lower than when looking at them in SCIEXOS.
Does anyone know the reason?
Cheers.
Cagakan

**Comment by PatricioMR** [comment]
Heyo, @Cagakan31 
Im working with a Sciex 5600 and I've noticed the same, but I made a analytical curve and got the same R^2. There's is no problem if it is proportional.


Matyash.
